src/org/ahmedbe00/qamp/zadaci/zadaca8/task1/Interfaces.java

```diff
@@ -0,0 +1,12 @@
+package org.ahmedbe00.qamp.zadaci.zadaca8.task1;
+
+public class Interfaces {
+
+    protected interface Flight {
+        String Fly();
+    }
+
+    protected interface Swim {
+        String Swim();
+    }
+}
```

Comment:
don't mix things. There is no need for these interfaces to be part of the same class, not even any class at all. In programming we prefer separation of concerns.